### ISSUE TITLE
Firefox RemoteTrackStats is Empty

### ISSUE THREAD

**Issue by themroberts** [twilio/twilio-video.js#295]
- [ x ] I have verified that the issue occurs with the latest twilio-video.js release and is not marked as a known issue in the [CHANGELOG.md](https://github.com/twilio/twilio-video.js/blob/master/CHANGELOG.md).
 - [ x ] I reviewed the [Common Issues](https://github.com/twilio/twilio-video.js/blob/master/COMMON_ISSUES.md) and open GitHub issues and verified that this report represents a potentially new issue.
 - [ x ] I verified that the [Quickstart](https://github.com/twilio/video-quickstart-js) application works in my environment.

**Code to reproduce the issue:**

1. Using the Quickstart Example. 
2. Join 2 users to a single room. 
3. After both have joined, using Firefox open console and call room GetStats
```window.room.getStats()```

Note* Chrome works as expected.

**Expected behavior:**
```
{
  localAudioTrackStats: [LocalAudioTrackStats]
  localVideoTrackStats: [LocalVideoTrackStats]
  peerConnectionId: "foobar"
  remoteAudioTrackStats:[RemoteAudioTrackStats]
  remoteVideoTrackStats:[RemoteVideoTrackStats]
}
```

**Actual behavior:**
```
{
  localAudioTrackStats: [LocalAudioTrackStats]
  localVideoTrackStats: [LocalVideoTrackStats]
  peerConnectionId: "foobar"
  remoteAudioTrackStats: [],
  remoteVideoTrackStats: []
}
```

**Software versions:**
 - [ x ] Browser(s): FireFox
 - [ x ] Operating System: OSX
 - [ x ] twilio-video.js: 1.8.0
 - [ x ] Third-party libraries (e.g., Angular, React, etc.): React

**Comment by markandrus** [comment]
Tackling in this upcoming sprint as JSDK-1927

**Comment by markandrus** [comment]
This was fixed in 1.10.0
